_rules/letter-spacing-not-important-24afc2.md

```diff
@@ -117,7 +117,7 @@ This `p` element has two [declared][] values for its `letter-spacing` property.
 
 #### Passed Example 6
 
-This `p` element has two [declared][] values for its `letter-spacing` property. The one from the style sheet wins the [cascade sort][] because it is [important][]. Since it is not [declared][] via the `style` attribute, it matches the **cascade** condition. Note that neither the **above minimum**, nor the **not important** conditions are matched. Styles with [author origin][] declared by assistive technologies may override this style. This is nonetheless bad practice and sufficient spacing should be used.
+This `p` element has two [declared][] values for its `letter-spacing` property. The one from the style sheet wins the [cascade sort][] because it is [important][]. Since it is not [declared][] via the `style` attribute, it matches the **cascade** condition. Note that neither the **wide enough**, nor the **not important** conditions are matched. Styles with [author origin][] declared by assistive technologies may override this style. This is nonetheless bad practice and sufficient spacing should be used.
```

Comment:
> This, and several other descriptions are still referencing the expectation, assuming the reader will know them. Please explain what about this passes, instead of having call-backs to the expectation.

Not referring to the labels of listed conditions is in direct contradiction with our rule design document.
https://act-rules.github.io/pages/design/rule-design/#listed-conditions (last but one bullet)
> Refer to the labels of the condition in the test case description to make the link explicit.